options.go

```diff
@@ -174,6 +174,14 @@ func ConfigureHelp(options HelpOptions) Option {
 	})
 }
 
+// ConfigureCompletion sets options for shell completion.
```

Comment:
Don't worry about having this function, just rely on the method on CompletionOptions. The only reason it exists for other things is legacy.